### ISSUE TITLE
Can you supply the generated thermal dataset？

### ISSUE THREAD

**Issue by Tianlu-Zhang** [zhanglichao/generatedTIR_tracking#5]
I tried to use your way to generate  thermal images,but I cannot get the results like the results in the paper. So do you have a plan to open the generated dataset,thanks for your answers.
![1](https://user-images.githubusercontent.com/48719188/67186377-8fc6ac80-f41a-11e9-9766-df2ea75d9e19.png)
And do you use this code to generate images:https://github.com/junyanz/pytorch-CycleGAN-and-pix2pix

**Comment by zhanglichao** [comment]
Hi,

Yes, the visualization analysis shows on the paper is evaluated on the testing dataset, very similar distribution with the training data (i.e. KAIST dataset), which explicitly help us select a better translation model to transfer large-scale RGB tracking datasets to TIR modality. The models fine-tuned with generated training data does improve the TIR tracking performance.

Thanks for your attention.

/Lichao

**Comment by ghost** [comment]
Hi, thank you for sharing. I downloaded your weight file. But I encountered a problem using https://github.com/junyanz/pytorch-CycleGAN-and-pix2pix. CycleGAN can generate thermal infrared images, but pix2pix reports an error. I enter the command in the terminal as: python test.py --dataroot ./datasets/visible/ --direction AtoB --model test --name net_G_pix2pix. But the error message is AttributeError: 'Sequential' object has no attribute 'model'.

**Comment by laisimiao** [comment]
Yes, opening your generated TIR dataset is beneficial for community research. For our convenience, could you upload and share your data?
